build/remote-config/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   // Note: settings in this file will override user-level settings in the workspace.
   //       Any setting that may be changed by the user should not be set here.
-  "git.defaultCloneDirectory": "/projects/"
+  "git.defaultCloneDirectory": "/projects/",
+  "terminal.integrated.cwd": "/projects/"
```

Comment:
@vitaliy-guliy 
could you test the use case when a workspace is not empty, so contains a project

I guess the `/projects` will be a directory in the terminal

Reply:
This pull request is created exactly to test the case.

Reply:
Have just tested the case. The settings force the terminal always switch to `/projects`, even if the project is opened.

Reply:
> This pull request is created exactly to test the case

you are working on the issue related to an empty workspace
so, I thought that you are going to test it for an empty workspace
I proposed to test it for another use case to avoid a regression...

Reply:
You can create a directory with `mkdir /protects/test-dir`, goto `Open Folder` and pick `/projects/test-dir`.
Then open a terminal and check the working directory.
It will be always `/projects` despite your workspace is not empty.

Reply:
Seems https://github.com/che-incubator/che-code/pull/328 is a solution for us